triple-reads-backend/index.ts

```diff
@@ -201,6 +211,183 @@ app.get('/books/:isbn', async (req: Request, res: Response) => {
   }
 });
 
+//POST TO INSERT
+app.post('/books', async (req: Request, res: Response) => {
+  try {
+    const { title, isbn, datePublished, abstract, image, authorNames, genreNames, publisher, admin, dateAdded } = req.body;
```

Comment:
I think it is more secure if the `dateAdded` is created by the backend and not passed in from frontend

Reply:
The dateAdded has not been removed from the request